### PR TITLE
Updating home page content

### PR DESCRIPTION
**Pull request by jennydove**
Trying to get homepage content to be a little more relevant.

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -5,13 +5,29 @@ draft: false
...
+Founded in April 2020, the TCN Coalition is a global community of technologists united by the mission to support exposure notification apps during the COVID-19 pandemic. The TCN Coalition believes that exposure notification apps must be interoperable, trusted by the public, and privacy-preserving to quickly and safely reduce the need for lockdowns. 
 
-Our first and foremost goal is to get secure tracing apps running on billions of users’ devices globally — fast. The TCN Coalition’s easy-to-implement privacy-first protocol, agreed upon and reviewed by dozens of experts, is open-source, extensible, free of charge, and available for implementation immediately. TCN — the core of the protocol — stands for temporary contact number. This completely anonymous number is generated to privately record interactions between compatible mobile devices without allowing them to be tracked.
+We achieve this mission through several programs: connecting and educating app developers about navigating the public health environment, assisting public health agencies in understanding how to deploy this technology safely and effectively, and supporting exposure notification apps with open source software.
```
Comment (danamlewis) [inline content/_index.md:12]: I'm not sure I would call these 'programs' - efforts maybe?

File: content/_index.md [hunk 2]
```diff
@@ -5,13 +5,29 @@ draft: false
...
 
-We urge all major technology companies, app developers and governments to implement apps compatible with this shared protocol. We welcome other groups engaged in similar efforts to join us at [TCN-coalition.org](http://TCN-coalition.org). We also encourage any other contact tracing coalitions to adopt similarly secure privacy-first protocols.
+## The Challenge
+
+A large number of organizations have sprung up to either enable or build out exposure notification applications since the start of the Covid-19 pandemic.
+
+These organizat ions span the globe and many of them have overlapping missions and work. On top of that, they are largely focused on their own product and are not thinking about the broader ecosystem, including how interoperability between these systems might work.
```
Comment (danamlewis) [inline content/_index.md:18]: is this intended to describe TCN members, or those who are not members? unclear.

File: content/_index.md [hunk 3]
```diff
@@ -5,13 +5,29 @@ draft: false
...
 
-We urge all major technology companies, app developers and governments to implement apps compatible with this shared protocol. We welcome other groups engaged in similar efforts to join us at [TCN-coalition.org](http://TCN-coalition.org). We also encourage any other contact tracing coalitions to adopt similarly secure privacy-first protocols.
+## The Challenge
+
+A large number of organizations have sprung up to either enable or build out exposure notification applications since the start of the Covid-19 pandemic.
+
+These organizat ions span the globe and many of them have overlapping missions and work. On top of that, they are largely focused on their own product and are not thinking about the broader ecosystem, including how interoperability between these systems might work. 
+
+The TCN Coalition serves as a gathering place for the overall effort. We increase visibility across the ecosystem, allowing for better communication, more effective use of resources, and faster delivery of effective exposure notification solutions as a result.
```
Comment (danamlewis) [inline content/_index.md:20]: "The TCN Coalition serves as a gathering place to increase visibility across the ecosystem, allowing for better communication, more effective use of resources, and faster delivery of effective exposure notification solutions as a result." (I would merge these two sentences)

File: content/_index.md [hunk 4]
```diff
@@ -5,13 +5,29 @@ draft: false
...
-We urge all major technology companies, app developers and governments to implement apps compatible with this shared protocol. We welcome other groups engaged in similar efforts to join us at [TCN-coalition.org](http://TCN-coalition.org). We also encourage any other contact tracing coalitions to adopt similarly secure privacy-first protocols.
+## The Challenge
+
+A large number of organizations have sprung up to either enable or build out exposure notification applications since the start of the Covid-19 pandemic.
+
+These organizat ions span the globe and many of them have overlapping missions and work. On top of that, they are largely focused on their own product and are not thinking about the broader ecosystem, including how interoperability between these systems might work. 
+
+The TCN Coalition serves as a gathering place for the overall effort. We increase visibility across the ecosystem, allowing for better communication, more effective use of resources, and faster delivery of effective exposure notification solutions as a result. 
+
+## Our Services
```
Comment (danamlewis) [inline content/_index.md:22]: Maybe 'Services TCN Coalition provides'?

File: content/_index.md [hunk 5]
```diff
@@ -5,13 +5,29 @@ draft: false
...
+
+> - **Affiliate Services.** We support all privacy-preserving exposure notification apps, no matter which protocol they are being built upon. We do this by providing education, networking, and expert advice to member teams looking to build out exposure notification apps.
```
Comment (danamlewis) [inline content/_index.md:24]: stronger language would help here, edit to "We provide education.."

File: content/_index.md [hunk 6]
```diff
@@ -5,13 +5,29 @@ draft: false
...
+
+These organizat ions span the globe and many of them have overlapping missions and work. On top of that, they are largely focused on their own product and are not thinking about the broader ecosystem, including how interoperability between these systems might work. 
+
+The TCN Coalition serves as a gathering place for the overall effort. We increase visibility across the ecosystem, allowing for better communication, more effective use of resources, and faster delivery of effective exposure notification solutions as a result. 
+
+## Our Services
+
+> - **Affiliate Services.** We support all privacy-preserving exposure notification apps, no matter which protocol they are being built upon. We do this by providing education, networking, and expert advice to member teams looking to build out exposure notification apps. 
+> - **Public Health Department Education.** We help public health organizations navigate this space through education, networking, and expert advice on exposure notification apps. 
+> - **Exposure Notification App Support.** Providing open source solutions to help exposure notification apps build safe and secure products.
```
Comment (danamlewis) [inline content/_index.md:26]: "Our Coalition members are contributing open source solutions, ranging from backend reference implementations to frontend app code that can be used by anyone."

**Review by danamlewis — COMMENTED** [review]
A few suggestions.

File: content/_index.md (hunk 2 above)
Comment (jennydove) [reply content/_index.md:18]: thanks. have clarified.

File: content/_index.md (hunk 6 above)
Comment (jennydove) [reply content/_index.md:26]: oh nice!

**Pull request merged**